code/datums/mind.dm

```diff
@@ -257,7 +257,7 @@
 
 		/** VAMPIRE ***/
 		text = "vampire"
-		if (ticker.mode.config_tag=="vampire")
+		if (ticker.mode.config_tag=="vampire" || ticker.mode.config_tag=="traitorvamp")

```

Comment:
# GAMEMODE CODE EVERYONE!
Isn't it fucking ~~atrocious~~ ~~blatantly ignoring all object oriented programming techniques~~ BEAUTIFUL?